ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -256,7 +257,7 @@ Load Expected Data Of RHODS Explore Section
 
 Wait Until Cards Are Loaded
     [Documentation]    Waits until the Application cards are displayed in the page
-    Run Keyword And Continue On Failure    Wait Until Page Contains Element
+    Run Keyword And Ignore Error    Wait Until Page Contains Element
```

Comment:
If we ignore the Error we might not see issues with the Dashboard page, cards are supposed to be loaded.

Reply:
But apparently `Run Keyword And Continue On Failure` blocks Test Setup (and everything afterwards).
Shouldn't we ignore it (RHOAIENG-404) and just warn.
It will fail later if there's anything other than  RHOAIENG-404. 
We can also refresh if you think that is a better approach ?

Reply:
if it's "Continue on Failure" it should not block test setup and everything afterwards, unless the reason for the subsequent failures is the same which caused the first failure. no?
If you think the waiting is not needed in a particular test case or step, you could set `wait_for_cards=${FALSE}` in the `Wait for RHODS Dashboard to Load` kw where `Wait Until Cards Are Loaded` is used.

I personally would not ignore the failure. The warn can easily get unseen in this case since the kw is mainly used nested in others

Reply:
ah ok that's normal, true. I thought it didn't continue other test setup keywords, sorry.

Anyways, instead of ignoring always, I would rather skip the wait with the existing parameter if needed. Ignoring this error doesn't sound good to me.

Reply:
I've implemented option (3) so it retries to reload page up to 3 times before giving up